app-text/libgepub/libgepub-0.7.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson xdg
+
+DESCRIPTION="GObject based library for handling and rendering epub documents"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libgepub"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="-widget +introspection"
+
+RDEPEND="
+	app-arch/libarchive
+	dev-libs/glib:2
+	dev-libs/libxml2
+	net-libs/libsoup:3.0
+	x11-libs/gtk+:3
```

Comment:
Existing problem, but this is only used as a test dependency.

Reply:
Ah, and the `tests` directory builds a program but doesn't actually run it as part of `meson test`.